how-tos/relpro.md

```diff
@@ -156,50 +156,87 @@ $ python releasetasks_graph_gen.py --release-runner-ini=../../../release-runner.
 $ python releasetasks_graph_gen.py --release-runner-ini=../../../release-runner.ini --branch-and-product-config=/home/cltbld/releasetasks/releasetasks/release_configs/prod_mozilla-esr52_firefox_rc_graph_2.yml --common-task-id=$TASK_TASKID_FROM_GRAPH1
 ```
 
+## 3. signoffs
 
-## 3. publish release
+### why
+* to guard against bad actors and compromised credentials we require that any changes to primary release channels (aurora, beta, release, esr) in balrog are signed off on by at least two people.
+
+### when
+
+* after the scheduled change has been created, and prior to the desired publish time
```

Comment:
might be my own ignorance but perhaps describe here how releng gets notified of balrog signoff requests?